x-pack/packages/kbn-elastic-assistant/impl/connectorland/connector_setup/index.tsx

```diff
@@ -72,12 +61,11 @@ export const useConnectorSetup = ({
   onSetupComplete,
   refetchConnectors,
 }: ConnectorSetupProps): {
-  connectorDialog: React.ReactElement;
-  connectorPrompt: React.ReactElement;
+  comments: EuiCommentProps[];
```

Comment:
appreciate the change in typing from `React.ReactElement` to `EuiCommentProps[]`